.github/workflows/clippy.yml

```diff
@@ -0,0 +1,26 @@
+on: [push]
+
+name: Clippy
+
+jobs:
+  clippy_check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          components: clippy
+          override: true
+
+      - name: Install gtk3 and dependencies.
+        run: sudo apt update && sudo apt install libcurl4-openssl-dev libelf-dev libdw-dev binutils-dev libgtk-3-dev libmount-dev libsoup2.4-dev libxtst-dev at-spi2-core openbox libxdo-dev
+        
+      - name: Run Clippy
+        uses: actions-rs/clippy-check@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Is it really needed?

Reply:
The token argument? Yes, since it's needed to post the output of clippy into Pull Requests and even the Actions summary page, at least for this Action. See here: https://github.com/actions-rs/clippy-check#inputs